go/bundle/bundle.go

```diff
@@ -0,0 +1,237 @@
+package bundle
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/WICG/webpackage/go/signedexchange"
+	"github.com/WICG/webpackage/go/signedexchange/cbor"
+)
+
+var HeaderMagicBytes = []byte{0x84, 0x48, 0xf0, 0x9f, 0x8c, 0x90, 0xf0, 0x9f, 0x93, 0xa6}
```

Comment:
`headerMagicBytes` to be unexported?

If this is used only in `WriteBundle`, I'd move there.

Reply:
The magic bytes are useful by itself. The clients can see if a given binary is a bundle or not.

Reply:
I'm against exposing a `[]byte`.

> The clients can see if a given binary is a bundle or not.

Would it be possible to add a function to do that?

Reply:
Discussed offline, and we compromised this since there is no good substitution.